### PR TITLE
Add xdotool

### PR DESCRIPTION
**Pull request by satmandu**
As part of trying to get x11 app suspend working...

Works properly:
- [x] x86_64

### CODE REVIEW

File: packages/xdotool.rb [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  depends_on 'libxkbcommon'
+
+  def self.build
+    system "PREFIX=#{CREW_PREFIX} INSTALLLIB=#{CREW_LIB_PREFIX} make WITHOUT_RPATH_FIX=1"
+  end
+
+  def self.install
+    system "LDCONFIG=#{CREW_PREFIX}/sbin/ldconfig DESTDIR=#{CREW_DEST_DIR} PREFIX=#{CREW_PREFIX} INSTALLLIB=#{CREW_LIB_PREFIX} make install"
+    # Remove execute bit from header file'
+    system "chmod -x #{CREW_DEST_DIR}#{CREW_PREFIX}/include/xdo.h"
```
Comment (uberhacker) [inline packages/xdotool.rb:24]: You can replace `#{CREW_DEST_DIR}#{CREW_PREFIX}` with `#{CREW_DEST_PREFIX}`.

File: packages/xdotool.rb [hunk 2]
```diff
@@ -0,0 +1,26 @@
+# Adapted from Arch Linux xdotool PKGBUILD at:
+# https://github.com/archlinux/svntogit-community/raw/packages/xdotool/trunk/PKGBUILD
+
+require 'package'
+
+class Xdotool < Package
+  description 'Command-line X11 automation tool'
+  version '3.20160805.1-3'
```
Comment (uberhacker) [inline packages/xdotool.rb:8]: Do we need the `-3`?

File: packages/xdotool.rb (hunk 1 above)
Comment (satmandu) [reply packages/xdotool.rb:24]: Noted.

File: packages/xdotool.rb (hunk 2 above)
Comment (satmandu) [reply packages/xdotool.rb:8]: no.

**Pull request merged**